### PR TITLE
At main page

### PR DESCRIPTION
**Pull request by sbfmede**

### CODE REVIEW

**Comment by sbfmede** [comment]
@varajan  could you please review PR. Thank you

**Review by varajan — COMMENTED** [review]
I've fixed found issues, please review my comments.

File: calculatorTest/MainPageFunctionalityTest.cs [hunk 1]
```diff
@@ -9,21 +9,17 @@
...
             browser = new ChromeDriver();
             browser.Url = (BASEURL + "Login");
-
             browser.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
             browser.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
-
-
             IWebElement loginfield = browser.FindElement(By.Id("login"));
             string name = "test";
             loginfield.SendKeys(name);
```
[the remark this answers is not in the document]
Comment (sbfmede) [reply calculatorTest/MainPageFunctionalityTest.cs:25]: that`s great point..thank you..I`ll correct

**Comment by sbfmede** [comment]
@varajan  hello!
thank you for the fix. 
Could you please review a new PR?
Thank you

File: calculatorTest/MainPageFunctionalityTest.cs [hunk 2]
```diff
@@ -33,63 +29,59 @@ public void Setup()
...
 
         [Test]
 
         public void Bounjdary_values_of_Rate_101_percent()
         {
-        browser.FindElement(By.Id("percent")).SendKeys("101");
-
+            browser.FindElement(By.Id("percent")).SendKeys("101");
             string percent = browser.FindElement(By.Id("percent")).GetAttribute("value");
             Assert.AreNotEqual("101", percent);
```
[the remark this answers is not in the document]
Comment (sbfmede) [reply calculatorTest/MainPageFunctionalityTest.cs]: > Maybe it's better to compare with expected value? What if the application will change '101' to '11'? The test will pass, but would it be correct application behavior?
> e.g. Assert.AreEqual("100")? Or some other expected value.

Thanks! that's a great point! I appreciate it

File: calculatorTest/GeneralSettings.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace CalculatorTest
+{
+    public static class GeneralSettings
+    {
+        public const string BASEURL = "http://localhost:64177/";
+
+    }
```
Comment (varajan) [inline calculatorTest/GeneralSettings.cs:11]: Please remove empty line.

```suggestion
        public const string BASEURL = "http://localhost:64177/";
    }
```

File: calculatorTest/LoginPageTest.cs [hunk 4]
```diff
@@ -8,16 +9,13 @@ namespace calculatorTest
...
-        const string BASEURL = "http://localhost:64177/";
         IWebDriver browser;
 
         [SetUp]
         public void Setup()
         {
             browser = new ChromeDriver();
-            browser.Url = (BASEURL + "Login");
-
+            browser.Url = (GeneralSettings.BASEURL + "Login");
```
Comment (varajan) [inline calculatorTest/LoginPageTest.cs:18]: Not an issue, just for your information: you don't need to take it in brackets.

```suggestion
            browser.Url = GeneralSettings.BASEURL + "Login";
```

File: calculatorTest/MainPageFunctionalityTest.cs [hunk 5]
```diff
@@ -9,87 +10,108 @@
...
-            loginfield.SendKeys(name);
-            IWebElement passwordfield = browser.FindElement(By.Id("password"));
-            passwordfield.SendKeys("newyork1");
+            //IWebElement loginfield = browser.FindElement(By.Id("login"));
+            browser.FindElement(By.Id("login")).SendKeys("test");
+            //string name = "test";
+            //loginfield.SendKeys(name);
+            //IWebElement passwordfield = browser.FindElement(By.Id("password"));
+            browser.FindElement(By.Id("password")).SendKeys("newyork1")
+            //passwordfield.SendKeys("newyork1");
```
Comment (varajan) [inline calculatorTest/MainPageFunctionalityTest.cs:30]: Don't save commented code. If you don't need it - remove.

File: calculatorTest/MainPageFunctionalityTest.cs [hunk 6]
```diff
@@ -9,87 +10,108 @@
...
             SelectElement year = new SelectElement(browser.FindElement(By.Id("year")));
             day.SelectByText("1");
             month.SelectByText("January");
             year.SelectByText("2010");
-
             browser.FindElement(By.Id("d365")).Click();
-
             string enddate = browser.FindElement(By.Id("endDate")).GetAttribute("value");
-            Assert.AreEqual("31/12/2010", enddate);//now we have some issue related to wrong end dates
+            Assert.AreEqual("01/01/2011", enddate);//now we have some issue related to wrong end dates
```
Comment (varajan) [inline calculatorTest/MainPageFunctionalityTest.cs:59]: I've fixed this bug (I think), please remove the comment. Pull changes to update local branch.

File: calculatorTest/MainPageFunctionalityTest.cs [hunk 7]
```diff
@@ -9,87 +10,108 @@
...
+        }
 
-    }
         [Test]
 
-        public void Boundary_values_of_Rate_0_percent()
+        public void Deposit_Amount_Valid_values_099()
         {
-            browser.FindElement(By.Id("percent")).SendKeys("-1");
+            browser.FindElement(By.Id("amount")).SendKeys("0.99"); //how we can set up a negative testing csenario
```
Comment (varajan) [inline calculatorTest/MainPageFunctionalityTest.cs:92]: I've fixed this bug (I think), please remove the comment. Pull changes to update local branch.

**Review by varajan — COMMENTED** [review]
Good, minor comments.

**Comment by sbfmede** [comment]
@varajan 
I have changed all items mentioned by you. Branch has merged

**Pull request merged**